libraries/Update/examples/HTTPS_OTA_Update/HTTPS_OTA_Update.ino

```diff
@@ -0,0 +1,92 @@
+#include "WiFi.h"
```

Comment:
A comment mentioning the details/purpose of the example?

Reply:
will you take care of adding a bit more info, like:
```cpp
// This sketch shows how to implement HTTPS firmware update Over The Air.
// Please provide your WiFi credentials, https URL to the firmware image and the server certificate.